gamemodes/terrortown/entities/weapons/weapon_ttt_physician_tracker.lua

```diff
@@ -0,0 +1,64 @@
+AddCSLuaFile()
+
+-- DEFINE_BASECLASS "weapon_tttbase"
+
+SWEP.HoldType               = "pistol"
+
+if CLIENT then
+   SWEP.PrintName           = "Health Tracker"
+   SWEP.Slot                = 8
+
+   SWEP.DrawCrosshair       = false
+   SWEP.ViewModelFlip       = false
+
+   SWEP.Icon                = "vgui/ttt/icon_flare"
+end
+
+SWEP.Base                   = "weapon_tttbase"
+
+SWEP.AutoSpawnable          = false
+
+SWEP.ViewModel              = Model("models/weapons/v_stunbaton.mdl")
+SWEP.WorldModel             = Model("models/weapons/w_stunbaton.mdl")
+
+SWEP.Primary.ClipSize       = -1
+SWEP.Primary.DefaultClip    = -1
+SWEP.Primary.Automatic      = false
+SWEP.Primary.Ammo           = "none"
+SWEP.Primary.Delay          = 0.75
+
+SWEP.Kind                   = WEAPON_EQUIP
+SWEP.InLoadoutFor           = {ROLE_PHYSICIAN}
+
+SWEP.AllowDelete            = false
+SWEP.AllowDrop              = false
+SWEP.NoSights               = true
+
+function SWEP:PlaceTracker()
+    self.Weapon:SetNextPrimaryFire( CurTime() + self.Primary.Delay )
+    self.Weapon:SetNextSecondaryFire( CurTime() + self.Primary.Delay )
+
+    if SERVER then
+        local ply = self:GetOwner()
+        local trace = ply:GetEyeTrace()
```

Comment:
Should probably check validity of `ply`. I've seen a few sporadic errors from weapons where the owner is invalid in cases where it was programmatically dropped or the player disconnected at the perfect time.